etl/steps/data/garden/cedlas/2024-03-08/sedlac.meta.yml

```diff
@@ -0,0 +1,749 @@
+          - "{definitions.welfare_equivalence}"
+        unit: '%'
+        short_unit: '%'
+        display:
+          name: <<table>> - Poverty gap index
+          <<: *common-display
+        presentation:
+          title_public: <<table>> - Poverty gap index
+
+      national_fgt2:
+        title: <<table>> - Poverty severity index
+        description_short: The squared poverty gap index (FGT(2)) is a poverty measure that reflects both the prevalence, the depth and the distribution of poverty. A poverty severity of 5 percent means that the poorest people are 5 percent worse off compared to poor people on average.
+        description_key:
+          - "{definitions.poverty_lines}"
+          - "{definitions.ppps}"
+          - "{definitions.welfare_equivalence}"
+        unit: '%'
+        short_unit: '%'
+        display:
+          name: <<table>> - Poverty severity index
+          <<: *common-display
+        presentation:
+          title_public: <<table>> - Poverty severity index
+
+      gini:
+        title: Gini coefficient (<<table>>)
+        description_short: The [Gini coefficient](#dod:gini) measures inequality on a scale from 0 to 1. Higher values indicate higher inequality.
+        description_key:
+          - "{definitions.welfare_equivalence}"
+        unit: ''
+        short_unit: ''
+        display:
+          name: Gini coefficient (<<table>>)
+          <<: *common-display
+        presentation:
+          title_public: "Income inequality: Gini coefficient in Latin America"
+
+      theil:
+        title: Theil index (<<table>>)
+        description_short: The Theil index is part of a larger family of measures referred to as the General Entropy class of inequality measures. It measures an entropic distance the population is away from the ideal egalitarian state of everyone having the same income.
+        description_key:
+          - "{definitions.welfare_equivalence}"
+        unit: ''
+        short_unit: ''
+        display:
+          name: Theil index (<<table>>)
+          <<: *common-display
+        presentation:
+          title_public: Theil index (<<table>>)
+
+      cv:
+        title: Coefficient of variation (<<table>>)
+        description_short: The coefficient of variation is a measure of inequality. It is the ratio of the standard deviation to the mean. Higher values indicate higher inequality.
+        description_key:
+          - "{definitions.welfare_equivalence}"
+        unit: ''
+        short_unit: ''
+        display:
+          name: Coefficient of variation (<<table>>)
+          <<: *common-display
+        presentation:
+          title_public: Coefficient of variation (<<table>>)
+
+      a_05:
+        title: Atkinson index - ε = 0.5 (<<table>>)
+        description_short: The [Atkinson index](#dod:atkinson) measures inequality on a scale from 0 to 1. Higher values indicate higher inequality. This version of the Atkinson index uses an inequality aversion parameter of 0.5.
```

Comment:
I am not sure you need this bit "This version of the Atkinson index uses an inequality aversion parameter of 0.5." in description_short as very few people would know what it means :)

Reply:
but I guess you do need to mention it so maybe explain what the parameter means  - "This parameter reflects the degree of concern for income disparities.."

Reply:
I added both the parameter and its description to description_key, thanks!